workspace/update-injected-packages/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "@pnpm/workspace.update-injected-packages",
```

We definitely should use the existing term "injected deps". Not sure if we want to use "update" for this as it usually means updating to a new version. I've seen syncing mentioned a lot, so maybe that could be used.

```suggestion
  "name": "@pnpm/workspace.injected-deps-syncer",
```

---

I didn't notice this comment, so I already renamed it to `@pnpm/workspace.sync-injected-deps`.

---

While I do like the sound of it, the convention currently in the repository is to use nouns for the package name, so `@pnpm/workspace.injected-deps-syncer` makes more sense.

---

https://github.com/pnpm/pnpm/pull/9100/commits/a9e6d3b674117cfbac434a2975e82ce7461497b3